code/game/gamemodes/vampire/vampire_gamemode.dm

```diff
@@ -22,7 +24,7 @@
 
 	var/list/datum/mind/possible_vampires = get_players_for_role(ROLE_VAMPIRE)
 
-	var/vampire_amount = 1 + round(num_players() / 10)
+	var/vampire_amount = 1 + (round(num_players() / 10) * (1 - vampire_penalty))
```

Comment:
Shouldn't this be just `* vampire_penalty`

Reply:
....It should, yes.